cmd/sealer/cmd/image/utils.go

```diff
@@ -0,0 +1,24 @@
+// Copyright © 2022 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package image
+
+// TODO we need to hava a rootfs interface
+func appRootPath() string {
+	return "application/apps"
```

Comment:
Should we make those path as pubic const ？

Reply:
we'd better have a rootfs manager instead of public const.